tests/test_flows.py

```diff
@@ -167,6 +167,51 @@ def test_ap_config_bss_tear_down(self):
             if vap.ssid != b'N/A':
                 self.fail('Wrong SSID: {vap.ssid} instead torn down'.format(vap=vap))
 
+
+    def test_ap_config_bss_tear_down_cli(self):
+        # Same test as the previous one but using CLI instead of dev_send_1905
+
+        env.beerocks_cli_command('bml_clear_wifi_credentials {}'.format(env.agents[0].mac))
+        # TODO replace with bml_set_wifi_credentials
```

Comment:
I don't understand than why add this test now if you can't use the new APIs yet. Should this commit perhaps belong to the next PR?

Reply:
@arnout Please explain your point of view

Reply:
@tomereli should learn to read :-P The lines above and below test the new commands introduced by this PR (clear and update).

Reply:
My bad.